cmd/juju/commands/upgradejuju_test.go

```diff
@@ -183,7 +182,7 @@ var upgradeJujuTests = []struct {
 	currentVersion: "3.0.0-quantal-amd64",
 	agentVersion:   "3.0.0",
 	args:           []string{"--version", "3.2.0"},
-	expectErr:      "no tools available",
+	expectErr:      "no matching tools available",
```

Comment:
these are different errors - i am concerned the new code is not returning the original error